src/components/SecretsDeleteModal/SecretsDeleteModal.js

```diff
@@ -29,9 +29,17 @@ const SecretsDeleteModal = props => {
       onRequestSubmit={handleDelete}
       onRequestClose={handleClick}
     >
-      <p>
-        Are you sure you want to delete the secret <strong>{id}</strong>?
-      </p>
+      <p>Are you sure you want to delete the(se) secret(s)?</p>
+      <ul>
+        {toBeDeleted.map(secret => {
+          const { name, namespace } = secret;
+          return (
+            <li key={`${name}:${namespace}`}>
+              <strong>{secret.name}</strong>
```

Comment:
style the `li` rather than using `strong` here since `strong` implies importance / urgency which doesn't apply to these values